e2e/e2eApi.ts

```diff
@@ -39,7 +39,22 @@ export class E2EApi {
     return (await this.count(selector)) > 0
   }
 
-  public clickOn = async (selector: string) => await (await this.getElement(selector)).click()
+  public clickOn = async (selector: string, x?: number, y?: number) => {
+    if (x === undefined && y === undefined) {
+      const elementHandle = await this.getElement(selector)
+      await elementHandle.click()
+
+      return
+    }
+
+    const dimensions: { x: number; y: number } = await this.page.evaluate(s => {
+      const rect = document.querySelector(s).getBoundingClientRect()
+
+      return { x: Math.round(rect.left), y: Math.round(rect.top) }
+    }, selector)
+
+    await this.page.mouse.click(dimensions.x + x, dimensions.y + y)
```

Comment:
https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pageclickselector-options

> This method fetches an element with selector, scrolls it into view if needed, and then uses page.mouse to click in the center of the element. If there's no element matching selector, the method throws an error.

To make this test working I need a different behavior because the center of `overlay` is `content` and click will be ignored